cip-00XX.md

```diff
@@ -0,0 +1,73 @@
+    CIP: ??
+    Title: Updated AT fees
+    Author: frank_the_tank, jjos, burstjack
+    Comments-Summary: No comments yet.
+    Comments-URI: 
+    Status: Draft
+    Type: Hard Fork
+    Created: 2019-04-07
+
+
+## Motivation
+
+With Burst, smart contracts are implemented using Automated Transactions (AT), a technology created by the CIYAM developers. Automated Transactions are turing-complete and thus have a potentially infinite number of use cases. Beside this possible infinite number of use cases the current fee structure for AT on Burst-chain is counterproductive.
+To attract business and developers for an investment/involvement with the ATs on the Burstcoin blockchain the fee structure needs to be reviewed and changed.
+
```

Comment:
BURST is shipped with a smart contract implemented as Automated Transactions (AT). This AT system was developed by [CIYAM](http://ciyam.org/at/at_api.htm) and is turing-complete, thus having potentially infinite number of use cases. The concept of this virtual machine is time-tested and already proven on-chain with some complex examples; like the BURST-lottery: which was written with only 37 steps in CIYAM bytecode.

Beside this possible infinite number of use cases the current fee structure for AT on Burst-chain is counterproductive. To attract business and developers for an investment/involvement with the ATs on the Burstcoin blockchain the fee structure needs to be reviewed and changed.

Reply:
This text should remove the current Motivation text